test/e2e/policyrecommendation_test.go

```diff
@@ -255,10 +255,12 @@ func testPolicyRecommendationFailed(t *testing.T, data *TestData) {
 }
 
 // Example output:
-// apiVersion: crd.antrea.io/v1alpha1
-// kind: NetworkPolicy
-// metadata:
-//   name: recommend-allow-anp-fj3hd
+//
+//	apiVersion: crd.antrea.io/v1alpha1
```

Comment:
I assume this change is just for better formatting, as there is no such indentation in the output.

Reply:
it's mentioned in the PR description
https://tip.golang.org/doc/go1.19#go-doc